util.go

```diff
@@ -173,7 +172,7 @@ func tarFile(tarWriter *tar.Writer, dir, relPath string) error {
 
 	info, err := os.Stat(absPath)
 	if err != nil {
-		return errors.Wrapf(err, "stat %s", absPath)
+		return errors.Wrapf(err, "stat '%s'", absPath)
```

Comment:
This should probably be `"getting stat info for path '%s'`.

Reply:
Fixed.